pointcloud/basic_octree_test.go

```diff
@@ -557,8 +557,19 @@ func TestBasicOctreePointcloudIngestion(t *testing.T) {
 
 // Test the functionalities involved with converting a pcd into a basic octree.
 func TestReadBasicOctreeFromPCD(t *testing.T) {
```

Comment:
This test seems to check that all points in the pointcloud are also in the octree. Do you know why it passed previously? Does it fail now if the points are different?

Reply:
See conversation here: https://viaminc.slack.com/archives/C036N5GJFLM/p1678115565635339

Reply:
Just to confirm, without the fix, the test does fail on large input sizes? And we're now testing large enough input sizes that we would catch the regression?